sdk/communication/azure-communication-callingserver/src/main/java/com/azure/communication/callingserver/models/events/CallingServerEventBase.java

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.communication.callingserver.models.events;
+
+/**
+ * Base type for all calling events.
```

Comment:
is this used at all? Not all events are inheriting from it. This is confusing

Reply:
now all events made to inherit from it. Inheritance was removed accidentally during refactoring, now it is put back